### ISSUE TITLE
david -g Nothing shows

### ISSUE THREAD

**Issue by gluons** [alanshaw/david#105]
`david -g` Nothing shows.

When my local dependencies are up to date, it shows `All dependencies up to date`.
But when I use david on global, nothing shows.

If global dependencies are up to date, it should shows any message like `All dependencies up to date`.

Running on Windows 10 Pro 64-bits

**Comment by doug-wade** [comment]
I repro this

```
dougwade code/stratocacher ‹master*› » david update --global
dougwade code/stratocacher ‹master*› » npm outdated -g
Package                                 Current  Wanted  Latest  Location
ava                                      0.13.0  0.14.0  0.14.0  /usr/local/lib > ava
babel-eslint                              6.0.0   6.0.4   6.0.4  /usr/local/lib > babel-eslint
browserify                               13.0.0  13.0.1  13.0.1  /usr/local/lib > browserify
bunyan                                    1.8.0   1.8.1   1.8.1  /usr/local/lib > bunyan
child                                     1.0.3   0.0.3   0.0.3  /usr/local/lib > child
csso                                      1.4.1   2.1.1   2.1.1  /usr/local/lib > csso
depcheck                                  0.6.1   0.6.3   0.6.3  /usr/local/lib > depcheck
eslint                                   2.10.0  2.10.1  2.10.1  /usr/local/lib > eslint
eslint-config-xo-space                   0.11.0  0.13.0  0.13.0  /usr/local/lib > eslint-config-xo-space
eslint-plugin-import                      1.0.3   1.8.0   1.8.0  /usr/local/lib > eslint-plugin-import
eslint-plugin-react                       4.2.3   5.1.1   5.1.1  /usr/local/lib > eslint-plugin-react
example                                   0.0.1   0.0.0   0.0.0  /usr/local/lib > example
generator-generator                       1.0.1   1.2.0   1.2.0  /usr/local/lib > generator-generator
generator-gulp-plugin-boilerplate         1.5.1   1.6.0   1.6.0  /usr/local/lib > generator-gulp-plugin-boilerplate
generator-koa-angular                     0.3.4   0.4.3   0.4.3  /usr/local/lib > generator-koa-angular
generator-node                           1.10.0  1.10.2  1.10.2  /usr/local/lib > generator-node
gitbook-cli                               2.1.1   2.1.3   2.1.3  /usr/local/lib > gitbook-cli
grunt                                     0.4.5   1.0.1   1.0.1  /usr/local/lib > grunt
gulp                                      1.2.1   3.9.1   3.9.1  /usr/local/lib > gulp
gulp-cli                                  0.4.0   1.2.1   1.2.1  /usr/local/lib > gulp-cli
inquirer                                 0.12.0   1.0.2   1.0.2  /usr/local/lib > inquirer
lerna                              2.0.0-beta.5   1.1.2   1.1.2  /usr/local/lib > lerna
less                                      2.6.1   2.7.1   2.7.1  /usr/local/lib > less
n                                         2.0.2   2.1.0   2.1.0  /usr/local/lib > n
node-inspector                           0.12.3  0.12.8  0.12.8  /usr/local/lib > node-inspector
npm                                     2.14.12   3.9.0   3.9.0  /usr/local/lib > npm
nsp                                       2.3.1   2.4.0   2.4.0  /usr/local/lib > nsp
nyc                                       6.4.0   6.4.4   6.4.4  /usr/local/lib > nyc
phantom                                   0.8.0   2.1.2   2.1.2  /usr/local/lib > phantom
phantomjs                                 1.9.8   2.1.7   2.1.7  /usr/local/lib > phantomjs
pm2                                       1.1.2   1.1.3   1.1.3  /usr/local/lib > pm2
sinopia                             1.5.0-pre.1   1.4.0   1.4.0  /usr/local/lib > sinopia
snyk                                     1.14.0  1.14.3  1.14.3  /usr/local/lib > snyk
tap                                       5.7.0   5.7.1   5.7.1  /usr/local/lib > tap
xo                                       0.15.0  0.15.1  0.15.1  /usr/local/lib > xo
yo                                        1.7.0   1.8.1   1.8.1  /usr/local/lib > yo
yuidocjs                                 0.10.0  0.10.1  0.10.1  /usr/local/lib > yuidocjs
dougwade code/stratocacher ‹master*› » david -v
v7.0.2
```

**Comment by cwonrails** [comment]
Same problem here. OSX 10.11.5, node version v6.2.0, npm version 3.9.2.

**Comment by gluons** [comment]
@alanshaw It hasn't released, right?
I'll wait for this. Thank you.

**Comment by alanshaw** [comment]
Released this morning david@9.0.0
